app/services/malware_scanner.rb

```diff
@@ -20,6 +18,7 @@ def call
       uploader: uploader,
       virus_found: virus_found?,
       scan_result: scan_result.result,
+      scanner_working: scanner_down?,
```

Comment:
I'm a bit confused by the logic here. If `scanner_down?` returns true when the scanner is _unavailable_ then wouldn't this always set `scanner_working` to the wrong value?

I think this would be better with the parameter renamed to `scanner_down` to match the method - `scanner_down: scanner_down?`.

EDIT: I think the confusion is caused by an error in `scanner_down?`. See my comments below. I think the right solution is to change the name of that method to `scanner_working?` and then this line becomes `scanner_working: scanner_working?`